lms/tests/test_models.py

```diff
@@ -1,143 +1,251 @@
-import datetime
+import random
+import string
+from unittest import mock
 
-from lms.lmsdb.models import Comment, Exercise, Notification, Solution, User
-from lms.models import notifications
-from lms.lmstests.public.general import tasks as general_tasks
+import pytest
+from peewee import SqliteDatabase
+
+from lms.lmsdb.models import Exercise, Notification, Solution, User
+from lms.models import notifications, solutions
 from lms.tests import conftest
 
 
-class TestUser:
+celery_async = (
```

Comment:
Why you need this? celery_eager fixture already fixed that

Reply:
See explanation on no_celery